recipes/openalea-deploy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "openalea.deploy" %}
+{% set version = "2.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openalea/deploy/archive/v{{ version }}.tar.gz
+  sha256: a2163cd662f5f2eb2062abc58982201a0241c94d84d28bfdca5388a78ac6ef22
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
+  preserve_egg_dir: True
```

Comment:
This is unusual.  Why do you need this?

Reply:
Mainly because openalea.deploy extend setuptools eggs by creating namespaces and storing data.
Without that, we have collision with identical path used by different packages.

Reply:
I had this because openalea.deploy generates namespace package. And with conda, there is a file collision: see conda/conda-build#649

Reply:
@scopatz Do you need more information?

Reply:
nope thanks!